tutorials/13_diagnostics_simulation_based_calibration.ipynb

```diff
@@ -541,13 +984,74 @@
   },
   {
    "cell_type": "markdown",
+   "id": "c2b021c4-c1aa-433d-95c4-e09e926f172f",
    "metadata": {},
    "source": [
     "The above provides a visual representation of the cumulative density function (CDF) of `ranks` (blue and orange for each dimension of `theta`) with respect to the 95% confidence interval of a uniform distribution (grey).\n"
    ]
   },
   {
    "cell_type": "markdown",
+   "id": "b3f1a0cc-d83f-4f0c-b174-ca367aab7699",
+   "metadata": {},
+   "source": [
+    "## multi dimensional SBC\n",
+    "\n",
+    "So far, we have performed the SBC checks for each dimension of our parameters $\\theta$ separately. SBI offers a way to perform this check for all dimensions at once. For this, we need to scale up our toy example a bit. So we increase the problem dimensionality to `16`."
```

Comment:
Why so many dimensions? Would 2 or 3 would be enough?

Reply:
Shoot, copy and paste error.

Reply:
fixed with bc41b716